plugin/diagnostics.py

```diff
@@ -344,7 +344,7 @@ def end(self) -> None:
     def format_diagnostic(self, diagnostic: Diagnostic) -> str:
         location = "{:>8}:{:<4}".format(
             diagnostic.range.start.row + 1, diagnostic.range.start.col + 1)
-        lines = diagnostic.message.splitlines()
+        lines = diagnostic.message.splitlines() or [""]
```

Comment:
This seems appropriate to do. `"".splitlines() == []`, whereas `"x".splitlines() == ["x"]`. A follow-up step would be to consult the maintainers of HIE.